src/bzflag/InputMenu.cxx

```diff
@@ -346,6 +369,10 @@ void            InputMenu::resize(int _width, int _height)
     }
     if (BZDB.isTrue("allowInputChange"))
         activeInput->setIndex(0);
+
+    i = 9;
+    SceneRenderer* renderer = getSceneRenderer();
+    ((HUDuiList*)listHUD[i++])->setIndex(renderer->getMaxMotionFactor() + 11);
```

Comment:
why are you bothering with `i` here? If you want the 9th element, just get the 9th element.